docs/pbc/all/search/202400.0/base-shop/third-party-integrations/integrate-algolia.md

```diff
@@ -0,0 +1,708 @@
+---
+title: Integrate Algolia
+description: Find out how you can integrate Algolia into your Spryker shop
+template: howto-guide-template
+last_updated: Feb 23, 2023
+redirect_from:
+  - /docs/pbc/all/search/202307.0/third-party-integrations/integrate-algolia.html
+---
+
+This document describes how to integrate [Algolia](/docs/pbc/all/search/{{page.version}}/base-shop/third-party-integrations/algolia.html) into a Spryker shop.
+
+## Prerequisites
+
+Before you can integrate the Algolia app, make sure that your project is ACP-enabled. See [App Composition Platform installation](/docs/acp/user/app-composition-platform-installation.html) for details.
+
+The Algolia app requires the following Spryker modules:
+
+* `spryker/catalog: "^5.8.0"`
+* `spryker/catalog-extension: "^1.0.0"`
+* `spryker/catalog-price-product-connector: "^1.4.0"`
+* `spryker/category: "^5.11.0"`
+* `spryker/category-storage: "^2.5.0"`
+* `spryker/message-broker-aws: "^1.3.1"`
+* `spryker/price-product: "^4.40.0"`
+* `spryker/product: "^6.32.0"`
+* `spryker/product-approval: "^1.1.0"` (Optional)
+* `spryker/product-category: "^4.19.0"`
+* `spryker/product-extension: "^1.5.0"`
+* `spryker/product-image: "^3.13.0"`
+* `spryker/product-label "^3.8.0"`
+* `spryker/product-label-storage "^2.6.0"`
+* `spryker/product-review: "^2.9.0"`
+* `spryker/search: "^8.19.3"`
+* `spryker/search-http: "^1.0.0"`
+* `spryker/store: "^1.17.0"`
+* `spryker/merchant-product-offer: "^1.5.0"` (Marketplace only)
+* `spryker/merchant-product-offer-data-import: "^1.1.0"` (Marketplace only)
+* `spryker/merchant-product-offer-search: "^1.4.0"` (Marketplace only)
+* `spryker/price-product-offer-data-import: "^0.7.1"` (Marketplace only)
+* `spryker/product-offer: "^1.4.0"` (Marketplace only)
```

Comment:
We are waiting for a finalized version pending release of PBC-209 ([RG](https://release.spryker.com/release-groups/view/4974)).

Anton has requested that we also include versions from [this release](https://release.spryker.com/release/pull-request/10342?type=0&released=1) into the list.

Reply:
I have removed the package requirements list from the docs page and now refer to the catalog page instead (which lists the same versions).

We should still wait for the modules to be released so that the modules can be wired correctly.

Reply:
Requirements were moved to the manifest, so this is no longer a blocker.